### DISCUSSION TITLE
How to speed up loading / Why boot init loading very slow ?

### DISCUSSION THREAD

**Discussion by Alceatraz** [tauri-apps/tauri#10822] [Q&A]
Hello everyone :

I try to write some simple applicaton with Tauri. But after launch the released application, Immediately
I noticed the loading process is very slow:

Application window open is really fast, But the inner webview render delay about 2 or 3 second.
I thought it was the react or my init too slow. Then I wrote a page like this

```html
<html><head><style>body{background-color:red</style><body></body></html>
```

Event with this, Still cost 3 seconds to loading the webpage.

I tested it on i3-9100F i5-10400T 7800X3D, None of them can open in 1 second.

And I also try use splashscreen, But the splash sceen also need 3 seconds to loading html.

Question: Is there any way to speed up loading or a slash screen shows in 0.1 seconds? ( Visualy smooth, A white blank then turns into black background brings really terriable UX).

**Comment by vednig** [comment]
To get native like feel, there must be some optimizations?

**Reply by Alceatraz** [reply]
> To get native like feel, there must be some optimizations?

AFAIK, WebView2 init too slow, The terriable white blank is waiting for webview
